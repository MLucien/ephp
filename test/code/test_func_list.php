<?php

$functions = get_defined_functions();
if (count($functions['internal']) > 10) {
    print "great! number of internal functions is greater than 10\n";
}
if (count($functions['user']) == 0) {
    print "great! there are not user functions registered yet\n";
}
var_dump(function_exists("strlen"));
var_dump(function_exists("function_not_exists_never"));
