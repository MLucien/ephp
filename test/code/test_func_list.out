great! number of internal functions is greater than 10
great! there are not user functions registered yet
bool(true)
bool(false)
